project_gtd/project_gtd.py

```diff
@@ -1,156 +1,89 @@
 
-    _defaults = {
-        'context_id': _get_context
-    }
-
     _group_by_full = {
         'timebox_id': _read_group_timebox_ids,
     }
 
-    def copy_data(self, cr, uid, id, default=None, context=None):
-        if context is None:
-            context = {}
+    @api.model
+    def copy_data(self, default=None):
         if not default:
             default = {}
-        if not default.get('timebox_id'):    
+        if not default.get('timebox_id'):
             default['timebox_id'] = False
-        if not default.get('context_id'):    
+        if not default.get('context_id'):
             default['context_id'] = False
-        return super(project_task, self).copy_data(
-            cr, uid, id, default, context)
-
-    def next_timebox(self, cr, uid, ids, *args):
-        timebox_obj = self.pool.get('project.gtd.timebox')
-        timebox_ids = timebox_obj.search(cr, uid, [])
-        if not timebox_ids:
-            return True
-        for task in self.browse(cr, uid, ids):
-            timebox = task.timebox_id
-            if not timebox:
-                self.write(cr, uid, task.id, {'timebox_id': timebox_ids[0]})
-            elif timebox_ids.index(timebox) != len(timebox_ids)-1:
-                index = timebox_ids.index(timebox)
-                self.write(
-                    cr, uid, task.id, {'timebox_id': timebox_ids[index+1]})
-        return True
-
-    def prev_timebox(self, cr, uid, ids, *args):
-        timebox_obj = self.pool.get('project.gtd.timebox')
-        timebox_ids = timebox_obj.search(cr, uid, [])
-        for task in self.browse(cr, uid, ids):
-            timebox = task.timebox_id
-            if timebox:
-                if timebox_ids.index(timebox):
-                    index = timebox_ids.index(timebox)
-                    self.write(
-                        cr, uid, task.id,
-                        {'timebox_id': timebox_ids[index - 1]})
-                else:
-                    self.write(cr, uid, task.id, {'timebox_id': False})
-        return True
-
-    def fields_view_get(self, cr, uid, view_id=None, view_type='form',
-                        context=None, toolbar=False, submenu=False):
-        if not context:
-            context = {}
-        res = super(project_task, self).fields_view_get(
-            cr, uid, view_id, view_type, context,
-            toolbar=toolbar, submenu=submenu)
-        search_extended = False
-        timebox_obj = self.pool.get('project.gtd.timebox')
-        if (res['type'] == 'search') and context.get('gtd', False):
-            timeboxes = timebox_obj.browse(
-                cr, uid, timebox_obj.search(cr, uid, []), context=context)
+        return super(ProjectTask, self).copy_data(default)
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
+                        submenu=False):
+        res = super(ProjectTask, self).fields_view_get(
+            view_id, view_type, toolbar=toolbar, submenu=submenu)
```

Comment:
Arguments should be simplified with *args and **kwargs

Reply:
@Jerther sorry?

Reply:
Since all arguments are solely used in the super call, I think it'd be safer and cleaner to write:
```python
def fields_view_get(self, *args, **kwargs):
        res = super(ProjectTask, self).fields_view_get(*args, **kwargs)
```